invenio_accounts/templates/invenio_accounts/_macros.html

```diff
@@ -46,7 +46,11 @@
   <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
   {%- for field, errors in form.errors.items() %}
     {%- for error in errors %}
-      <p>{{error}}</p>
+        {%- if error=="The response parameter is missing." %}
```

Comment:
You can have a look at the structure of errors by using: ``{{form.errors}}``

Not completely sure but I think you can use:
```
{% if errors is mapping %}
{% endif %}
```

or 

```
{%- for item in sitemap recursive %}
```